### PR TITLE
Release 3.9.0 smks

### PR DESCRIPTION
**Pull request by kiranharidas187**

### CODE REVIEW

File: build_config [hunk 1]
```diff
@@ -41,11 +38,10 @@ cordova-plugin=cordova-plugin-telerik-imagepicker@2.3.3
 cordova-plugin=cordova-plugin-photo-library
```
Comment (swayangjit) [inline build_config:38]: Media and permission  is used for sunbird use cases

File: src/app/home/admin-home/admin-home.page.ts [hunk 2]
```diff
@@ -97,13 +102,17 @@ export class AdminHomePage implements OnInit, OnDestroy, OnTabViewWillEnter {
...
+        subType: 'project',
+        action: 'create'
+      };
+
+      this.formService.getForm(request).subscribe(result => {
         this.getTaskForm();
-        if (data.result && data.result.length) {
-          this.storage.setLocalStorage(localStorageConstants.PROJECT_META_FORM, data.result).then(resp => {
+        if (result.form.data.fields) {
+          this.storage.setLocalStorage(localStorageConstants.PROJECT_META_FORM, result.form.data.fields).then(resp => {
```
Comment (swayangjit) [inline src/app/home/admin-home/admin-home.page.ts:115]: Use formnFrameworkUtilService  like following
```
const personaTenantFormData: FieldConfig<any>[] = await this.formnFrameworkService.getFormFields(FormConstants.TENANT_PERSONAINFO,
      this.profile.rootOrg.rootOrgId);
```

File: src/app/home/admin-home/admin-home.page.ts [hunk 3]
```diff
@@ -119,9 +128,16 @@ export class AdminHomePage implements OnInit, OnDestroy, OnTabViewWillEnter {
...
+      const request: FormRequest = {
+        from: CachedItemRequestSourceFrom.SERVER,
+        type: 'user',
+        subType: 'project',
+        action: 'createTask'
+      };
+
+      this.formService.getForm(request).subscribe(result => {
+        if (result.form.data.fields) {
+          this.storage.setLocalStorage(localStorageConstants.TASK_META_FORM, result.form.data.fields).then(resp => {
```
Comment (swayangjit) [inline src/app/home/admin-home/admin-home.page.ts:140]: Same here use formnFrameworkUtilService

File: build_config (hunk 1 above)
Comment (kiranharidas187) [reply build_config:38]: There is duplicate plugins , hence deleted one

File: src/app/home/admin-home/admin-home.page.ts (hunk 2 above)
Comment (kiranharidas187) [reply src/app/home/admin-home/admin-home.page.ts:115]: We actually save it in local storage for offline usage off form. If the form is available in local storage we don't call the API

File: src/app/home/admin-home/admin-home.page.ts (hunk 3 above)
Comment (kiranharidas187) [reply src/app/home/admin-home/admin-home.page.ts:140]: We write the forms in to local storage so that we can use it in offline mode also

**Review by swayangjit — APPROVED** [review]

**Review by swayangjit — APPROVED** [review]

**Pull request merged**